src/components/SubtaskDropdownWithSearch/SubtaskDropdownWithSearch.tsx

```diff
@@ -56,13 +58,13 @@ export const SubtaskDropdownWithSearch: FC<DropdownWithSearchProps> = ({
   const [loading, setLoading] = useState<boolean>(false);
   const [isDirtySearch, setIsDirtySearch] = useState(false);
   const [searchQuery, setSearchQuery] = useState<string>("");
-  const [parents, setParents] = useState<Record<string, any>>([]);
+  const [parents, setParents] = useState<Record<SearchIssueItem["id"], SearchIssueItem>>({});
   const [parentOptions, setParentOptions] = useState<
-    DropdownValueType<any>[] | DropdownHeaderType[]
+    DropdownValueType<SearchIssueItem["id"]>[] | DropdownHeaderType[]
   >([]);
 
   const getIssueTitle = (): string => {
-    const issue = get(parents, [value], null);
+    const issue = get(parents, [value as string], null);
```

Comment:
Would be nice to remove `lodash` here so we keep our typings. Or at the very least, it should be `lodash-es` but ideally no `lodash`.

Reply:
rewrote to `lodash-es`